spec/ics-007-tendermint-client/README.md

```diff
@@ -214,8 +214,9 @@ function checkValidityAndUpdateState(
     // update latest timestamp
     clientState.latestTimestamp = header.timestamp
     // create recorded consensus state, save it
-    consensusState = ConsensusState{header.validatorSet, header.commitmentRoot, header.timestamp}
+    consensusState = ConsensusState{header.timestamp, header.validatorSet, header.commitmentRoot}
     set("clients/{identifier}/consensusStates/{header.height}", consensusState)
+    set("clients/{identifier}/processedTimes/{header.height}", currentTimestamp())
```

Comment:
Does it make sense adding `processedTime` as a field to the ConsensusState? It might be a bit easier to understand the semantic of `delayPeriod`?

Reply:
No, because we don't want to *store* it in the consensus state, which needs to be verified e.g. during the handshake